vnext/package.json

```diff
@@ -20,7 +20,8 @@
     "lint": "just-scripts lint",
     "start": "react-native start",
     "watch": "tsc -w",
-    "validate-overrides": "override validate ./src/overrides.json ./ReactCopies/overrides.json ./DeforkingPatches/overrides.json"
+    "validate-overrides": "override validate ./src/overrides.json ./ReactCopies/overrides.json ./DeforkingPatches/overrides.json",
+    "postinstall": "node scripts/postinstall.js"
```

Comment:
We will likely want to hook this into the build as well. I.e. for developers making JS changes in react-native-windows, we would want `yarn build` to lead to changes being reflected.